app/styles/pages/sunday.scss

```diff
@@ -1,95 +1,102 @@
 .bulletin {
   .cover {
-    background-image: url(images/theme-960.jpg);
+    background-image: url(images/theme-1920.jpg);
+    background-position: center;
     background-size: cover;
     box-shadow: inset 0 0 500px rgba(255, 255, 255, .5);
     position: relative;
     text-align: center;
 
     h1 {
+      margin: 0;
+    }
+
+    .name {
       font-family: 'Arapey', serif;
       font-style: italic;
       font-weight: 400;
-      margin: 0;
-
-      small {
-        font-family: sans-serif;
-        font-style: normal;
-        font-size: 18px;
-      }
     }
 
-    @media only screen and (min-width : 768px) {
-      text-align: left;
-      padding-left: 42px;
-      background-image: url(images/theme-1920.jpg);
-
-      h1 {
-        font-size: 60px;
-        line-height: 50px;
-
-        small {
-          font-size: 24px;
-        }
-      }
+    .description {
+      font-family: sans-serif;
+      font-style: normal;
+      font-size: 18px;
     }
   }
 
   .bulletin-contents {
     position: relative;
 
-    @media only screen and (min-width : 320px) {
-      section.announcements {
-        background-color: #FDF2FD;
-      }
+    section {
+      padding: 100px 0;
     }
+  }
+
+  .service-order {
+    ul {
+      list-style-type: none;
+      padding: 0;
+
+      li {
+        margin-bottom: 60px;
 
-    @media only screen and (min-width : 768px) {
-      .inner:before {
-        content: '';
-        position: absolute;
-        right: 0;
-        left: 50%;
-        top: 0;
-        bottom: 0;
-        background-color: #FDF2FD;
+        &:last-child {
+          margin-bottom: 0;
+        }
       }
     }
 
-    section {
-      padding: 100px 0;
+    h2 {
+      font-family: 'Arapey', serif;
+      font-style: italic;
```

Comment:
Properties should be ordered font-family, font-size, font-style, font-weight, margin-top